### PR TITLE
Update the configmap example with Koji's fast upload option

### PR DESCRIPTION
**Pull request by cverna**
Signed-off-by: Clement Verna <cverna@tutanota.com>

### CODE REVIEW

**Comment by twaugh** [comment]
That doesn't seem valid according to the [schema](https://github.com/containerbuildsystem/atomic-reactor/blob/master/atomic_reactor/schemas/config.json).

**Comment by cverna** [comment]
Coresponding PR --> https://github.com/containerbuildsystem/atomic-reactor/pull/1220

File: docs/build_parameters.rst [hunk 1]
```diff
@@ -83,6 +83,7 @@ Example of **REACTOR_CONFIG**::
         root_url: https://koji.example.com/root
         auth:
             ssl_certs_dir: /var/run/secrets/atomic-reactor/kojisecret
+        use_fast_upload: true
```
Comment (MartinBasti) [inline docs/build_parameters.rst]: should we recommend `True` here? Is that only current fedora-infra workaround or a preferred way of uploading?

Comment (cverna) [reply docs/build_parameters.rst]: I am honestly not sure, but `use_fast_upload` defaults to true in the koji.conf you get when installing the koji client. 
Generally I think setting it to true will not hurt :smile:, but I am happy to change that to `false` if needed.

Comment (twaugh) [reply docs/build_parameters.rst]: I'd actually be happy for us to default to fast-upload, i.e. for this option to have default value `true`.

Comment (MartinBasti) [reply docs/build_parameters.rst]: ACK

**Comment by twaugh** [comment]
Now the default is true, the only useful value in the config map is false. Should we change that in this example?

**Comment by cverna** [comment]
> Now the default is true, the only useful value in the config map is false. Should we change that in this example?

Updated :+1:

**Pull request merged**